targets/TARGET_Cypress/TARGET_PSOC6/TARGET_CY8CKIT_062_WIFI_BT_PSA/spm_api.h

```diff
@@ -0,0 +1,30 @@
+/* Copyright (c) 2017-2018 ARM Limited
```

Comment:
conflicts with `hal\spm_api.h` please remove

Reply:
Fixed: removed this header